www/src/components/pre.js

```diff
@@ -0,0 +1,51 @@
+import React from "react"
+import Highlight, { defaultProps } from "prism-react-renderer"
+
+import Copy from "./copy"
+import { space } from "../utils/presets"
+
+const getParams = (name = ``) => {
+  const [lang, params = ``] = name.split(`:`)
+  return [lang.split(`language-`).pop()].concat(
+    params.split(`&`).reduce((merged, param) => {
+      const [key, value] = param.split(`=`)
+      merged[key] = value
+      return merged
+    }, {})
+  )
+}
+
+export default ({ children }) => {
+  const [language, { title = `` }] = getParams(children.props.className)
+  const content = children.props.children
+  return (
+    <Highlight
```

Comment:
Note: we’ll probably want to pass our custom Prism theme as a theme here — will do just that!